confgenerator/testdata/valid/linux/logging-multiple_google_exporters/input.yaml

```diff
@@ -48,12 +48,13 @@ logging:
       time_key: time_key_5
       time_format: time_format_5
   exporters:
-    google:
-      type: google_cloud_logging
     google1:
```

Comment:
Do we absolutely need to support this use case for GA?

Reply:
Discussed offline. Yes we do need to support this unfortunately